### DISCUSSION TITLE
docs tutorial

### DISCUSSION THREAD

**Discussion by gisraelsplunk** [reduxjs/redux#4597] [Help] [answered]
hi why do we {/* omit form inputs */}  on this page? https://redux.js.org/tutorials/essentials/part-3-data-flow There is nothing to type in if we do that?

**Comment by markerikson** [comment, marked as answer]
@gisraelsplunk : The code snippets in the page are deliberately kept shorter in some cases to keep the page from getting too long.  If you want to see the full code, it's available in the CodeSandbox embeds on each page!
